packer/builder/azure/arm/step_ip_address.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
+
+package arm
+
+import (
+	"fmt"
+
+	"github.com/Azure/packer-azure/packer/builder/azure/common/constants"
+	"github.com/mitchellh/multistep"
+	"github.com/mitchellh/packer/packer"
+)
+
+type StepIPAddress struct {
```

Comment:
Done.  I choose 'Get'.  I'll change the Query (from step_query_vm) to Get as part of my list of TODOs.